src/EditorFeatures/CSharp/InlineRename/CSharpEditorInlineRenameService.cs

```diff
@@ -149,10 +153,13 @@ void AddSpanOfInterest(SourceText documentText, TextSpan fallbackSpan, TextSpan?
             var length = endPosition - startPosition;
 
             surroundingSpanOfInterest = new TextSpan(startPosition, length);
-
             if (seen.Add(surroundingSpanOfInterest.Value))
             {
-                resultBuilder.Add(documentText.GetSubText(surroundingSpanOfInterest.Value).ToString());
+                var filePath = documentText.GetDocumentWithFrozenPartialSemantics(cancellationToken)?.FilePath;
```

Comment:
Will it work if you direclty call https://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/Workspace/TextExtensions.cs,8844857d387b5686,references?